shell/platform/fuchsia/flutter/meta/flutter_runner_tests.cmx

```diff
@@ -10,7 +10,7 @@
     "services": [
       "fuchsia.accessibility.semantics.SemanticsManager",
       "fuchsia.intl.PropertyProvider",
-      "fuchsia.sys.Launcher"
+      "fuchsia.process.Launcher"
```

Comment:
Same question

Reply:
Yes, this definitely belongs as the tests assert/logspam on Fuchsia without it